src/core/ext/filters/http/server/http_server_filter.cc

```diff
@@ -266,49 +260,55 @@ static grpc_error* server_filter_incoming_metadata(grpc_call_element* elem,
   return error;
 }
 
-static void hs_on_recv(void* user_data, grpc_error* err) {
+static void recv_initial_metadata_ready(void* user_data, grpc_error* err) {
```

Comment:
Nit, but can you keep something in the func names that shows this to be from http server file? I have future plans to change all static functions to have unique names with a linter to enforce. 

flamegraphs do not show file metadata, so if we see a `recv_initial_metadata_ready`, we don't know which filter it came from

Reply:
Done.

Note that this problem will go away when we convert the filter API to C++, since the filter class will be part of the reported function name.